Sources/CardPayments/CardClient.swift

```diff
@@ -36,6 +36,13 @@ public class CardClient: NSObject {
     /// - Throws: PayPalSDK error if approve order could not complete successfully
     public func approveOrder(request: CardRequest) {
         Task {
+            do {
+                _ = try await apiClient.getClientID()
```

Comment:
I updated the docstring on `getClientID` to make it clear that the value is fetched from cache, if available. However, we _could_ update this method name to reflect that - if folks find that more clear. Open to name suggestions!

Reply:
Yeah, I feel like updating the name of the method would be appropriate. I can't think of a name that isn't just way too wordy though. 🤔 Something like `fetchNewOrCachedClientID` was my initial though but geez if that isn't an absolute mouthful.

Reply:
Haha right? I keep thinking of `fetchOrReturnRemoteConfiguration` on BT, which essentially is the same. But that method name was never intuitive to me either. I like your idea more, using the word `cached` is helpful!

How about `fetchCachedOrRemoteClientID`?

Reply:
I dig `fetchCachedOrRemoteClientID`!

Reply:
Done!